docs/proposals/external-ca.md

```diff
@@ -0,0 +1,112 @@
+# External Certificate Authority Proposal
+
+## Context
+
+To enable mTLS in the mesh, each dataplane has to receive a unique certificate. This certificate has to be signed by the root certificate which is unique for every mesh.
+Right now, Kuma auto-generates root certificate when mesh is created. Root certificates are stored in Postgres on Universal and as a Secret on Kubernetes.
+In many organisations there is already Certificate Authority, so there should be a way to use it with Kuma.
+
+## Requirements
+
+User wants to generate root certificate and provide it to Kuma.
+
+## Proposed configuration model
+
+```yaml
+type: Mesh
+name: default
+mtls:
+  ca:
+    external: {}
+  enabled: true
+```
+
+## How to provide certificates to Kuma
```

Comment:
Let's document requirements to the CA cert provided by a user (see https://github.com/spiffe/spiffe/blob/master/standards/X509-SVID.md#32-signing-certificates)